jobs/openshift4/cluster/create/Jenkinsfile

```diff
@@ -0,0 +1,146 @@
+#!groovy
+// by default build number is used as cluster to prevent clashes with possibly already existing clusters
+String clusterName = BUILD_NUMBER
+String awsRegion = params.AWS_REGION
+String baseDomain = "intlyqe.com"
+if (CLUSTER_NAME) {
+    clusterName = CLUSTER_NAME
+}
+String clusterInfoDirectory = "clusters/" + clusterName
+String installConfigDirectory = ""
+String rootWorkspaceDir = ""
+
+def err = null
+String adminPassword = ""
+
+node("cirhos_rhel7") {
+    timeout(120) { ansiColor("gnome-terminal") { timestamps {
+        try {
+            currentBuild.displayName = "${currentBuild.displayName} ${clusterName}"
+            currentBuild.description = "clusterName: ${clusterName}\nawsRegion: ${awsRegion}"
+            
+            stage("Configure aws cli") {
+                withCredentials([string(credentialsId: "qe-iam-access-key-id", variable: "AWS_ACCESS_KEY_ID")]) {
+                    withCredentials([string(credentialsId: "qe-iam-access-secret-key", variable: "AWS_SECRET_ACCESS_KEY")]) {                
```

Comment:
Why do we need 2 different credentials here? What type of credential are you using?

Reply:
IAM credentials - `aws_access_key_id` and `aws_secret_access_key`

Reply:
I mean what type of jenkins credential are you using? I believe if you were using an AWS credential you could do this:

```
withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', credentialsId: 'aws-credentials-id', accessKeyVariable: 'AWS_ACCESS_KEY_ID', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
    //Do stuff with creds
}
```

Reply:
Looks like we might need to install the aws credentials plugin, appears not to be installed.

Reply:
Tried it. does not work. will try to change whatever is currently used to cleaner solution